### ISSUE TITLE
admin:user:change-password on Magento 2.2 throws an error "Area code not set: Area code must be set before starting a session."

### ISSUE THREAD

**Issue by a-dite** [netz98/n98-magerun2#329]
reproduced on:

PHP 7.0.24
Magento 2.2 EE

**Comment by EliasKotlyar** [comment]
I suppose its a duplicate of:
https://github.com/netz98/n98-magerun2/issues/251

**Comment by watermark** [comment]
I do not think it's a duplicate.  I'm also getting this on Magento CE 2.2.3 using magerun 2.1.0.

**Comment by EliasKotlyar** [comment]
@watermark : Do you use any Extension? It may be related to some Events or Plugins. I checked on a Stock 2.2.3 and could not reproduce.

**Comment by yoosefi** [comment]
Vanilla installation of CE 2.2 here, getting this issue.

**Comment by yoosefi** [comment]
Vanilla CE 2.2.3 here, only base Magento modules. Getting this error.

**Comment by pmsteil** [comment]
I am also getting this error.  Base Magento CE v2.2.3 with one extension installed - the MGS Front End Builder (https://www.magesolution.com/mgs-frontend-builder-theme-for-magento2.html).

Any workaround available for this?  Really need to be able to set the password on a user for testing.

Thanks,
Patrick

**Comment by cooper85** [comment]
https://github.com/netz98/n98-magerun2/issues/381 is not fixed this issue
That fix for command customer:change-password, this issue related to command admin:user:change-password

**Comment by amenk** [comment]
I still have this error for some users ..., for example such I created via API
